contracts/upgradeable_contracts/BasicHomeBridge.sol

```diff
@@ -155,4 +173,41 @@ contract BasicHomeBridge is EternalStorage, Validatable {
 
     function onFailedAffirmation(address, uint256, bytes32) internal {
     }
+
+    function feeManagerContract() public view returns(address) {
+        return addressStorage[keccak256(abi.encodePacked("feeManagerContract"))];
+    }
+
+    function setFeeManagerContract(address _feeManager) public onlyOwner {
+        require(_feeManager == address(0) || isContract(_feeManager));
+        addressStorage[keccak256(abi.encodePacked("feeManagerContract"))] = _feeManager;
+    }
+
+    function setFee(uint256 _fee) external onlyIfOwnerOfProxy {
+        require(feeManagerContract().delegatecall(abi.encodeWithSignature("setFee(uint256)", _fee)));
+    }
+
+    function getFee() public view returns(uint256) {
```

Comment:
it makes sense to call the corresponding method of the fee manager contract here because it is responsible for storing the fee value (imagine if the value should be calculated by somehow)

Reply:
Thanks, fixed https://github.com/poanetwork/poa-bridge-contracts/pull/132/commits/8876e15fffecea4a5dcb6fc62d20c6057512a6cf